packages/js/msgpack/src/index.ts

```diff
@@ -1,10 +1,12 @@
-import {
-  Encoder,
-  Decoder,
-  ExtensionCodec,
-  encode,
-  decode,
-} from "@msgpack/msgpack";
+import { Encoder, Decoder, ExtensionCodec } from "@msgpack/msgpack";
+
+export function isBuffer(maybeBuf: unknown): maybeBuf is BufferSource {
```

Comment:
Where is this function being used? We have it defined in two places (https://github.com/polywrap/toolchain/blob/origin-dev/packages/js/core/src/utils/is-buffer.ts) I think we should have it in one place

Reply:
Will remove it from here not needed anymore